channels/tr.m3u

```diff
@@ -241,46 +207,38 @@ https://live.kanal7.com/live/kanal7LiveMobile/index.m3u8
 https://mn-nl.mncdn.com/blutv_kanal7/kanal7_sd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Kanal7AvrupaHD.tr" tvg-name="Kanal 7 Avrupa HD" tvg-country="TR" tvg-language="Turkish" tvg-logo="https://i.imgur.com/oHq0PKQ.jpg" group-title="",Kanal 7 Avrupa HD
 https://live.kanal7.com/live/kanal7AvrupaLive/index.m3u8
-#EXTINF:-1 tvg-id="KANAL7HD.tr" tvg-name="KANAL 7 HD" tvg-country="TR" tvg-language="" tvg-logo="" group-title="",KANAL 7 HD
-https://mn-nl.mncdn.com/blutv_kanal7/smil:kanal7_sd.smil/chunklist_b2064000.m3u8
```

Comment:
#EXTVLCOPT:http-user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36" is needed for this source